packages/providers/upload-aws-s3/src/utils.ts

```diff
@@ -5,6 +5,29 @@ interface BucketInfo {
   err?: string;
 }
 
+export function isUrlFromBucket(fileUrl: string, bucketName: string, bucketBaseUrl = '') {
```

Comment:
Shouldn't we specify the return type on functions? 🤔

Reply:
do we have a standard for it? I am normally used to let typescript infer the return type.

Reply:
doesn't hurt to add it here I imagine, I will add it!

Reply:
I think it was a subject in the TS workshop we had and we did not manage to a conclusion on that specific point 🤔